bootstrap/023_simple_materializations.sql

```diff
@@ -93,3 +94,42 @@ begin
     call internal.refresh_warehouses();
     return 'success';
 end;
+
+CREATE TABLE IF NOT EXISTS internal_reporting_mv.warehouse_load_history (start_time timestamp_ltz,
+        end_time timestamp_ltz,
+        warehouse_name text,
+        avg_running NUMBER(38,2),
+        avg_queued_load NUMBER(38,2),
+        avg_queued_provisioning NUMBER(38,2),
+        avg_blocked NUMBER(38,2));
+create or replace view reporting.warehouse_load_history as select * from internal_reporting_mv.warehouse_load_history;
+
+
+CREATE OR REPLACE PROCEDURE internal.refresh_one_warehouse_load_history(warehouse_name varchar) RETURNS table(varchar) LANGUAGE SQL
```

Comment:
I think you are accessing the varchar column from `RETURNS table(varchar)` as `sql` in `TASKS.WAREHOUSE_LOAD_MAINTENANCE`. Should this be `RETURNS table(sql varchar)` to be a little more obvious?  (tbh, it's not clear to me how this works to not give a column name when returning a table from a proc, but am assuming snowflake figures this out :D)

Reply:
weird, I put that somewhere but must have accidentally dropped it. Fixed